src/platform/packages/private/kbn-grid-layout/grid/types.ts

```diff
@@ -107,18 +107,24 @@ export interface PanelInteractionEvent {
   };
 }
 
-// TODO: Remove from Dashboard plugin as part of https://github.com/elastic/kibana/issues/190446
-export enum PanelPlacementStrategy {
-  /** Place on the very top of the grid layout, add the height of this panel to all other panels. */
-  placeAtTop = 'placeAtTop',
-  /** Look for the smallest y and x value where the default panel will fit. */
-  findTopLeftMostOpenSpace = 'findTopLeftMostOpenSpace',
-}
-
-export interface PanelPlacementSettings {
-  strategy?: PanelPlacementStrategy;
-  height: number;
-  width: number;
-}
```

Comment:
These types should have been removed ages ago - they were used back when we wanted the `GridLayout` package to contain panel placement logic, but we moved away from that approach quite a long time ago 🙈

Reply:
Wow that's an oldie! Good removal.